crypto/fipsmodule/sha/internal.h

```diff
@@ -80,7 +80,8 @@ struct keccak_st {
   size_t md_size;                                  // output length, variable in XOF (SHAKE)
   size_t buf_load;                                 // used bytes in below buffer
   uint8_t buf[SHA3_MAX_BLOCKSIZE];                 // should have at least the max data block size bytes
-  uint8_t pad;
+  uint8_t pad;                                     // padding character
+  uint8_t padded;                                  // denotes if padding has been performed (used to facilitate multiple squeezes)
```

Comment:
Seems like line will be too long.
```suggestion
  uint8_t padded;                                  // denotes if padding has been performed 
                                                   // (used to facilitate multiple squeezes)
```

Reply:
fixed in [3240910](https://github.com/aws/aws-lc/pull/1839/commits/32409105f83366884c41183fdd46d75fd57f2367) (removed surplus info)